src/controllers/bulk/bulkcontroller.cpp

```diff
@@ -299,11 +304,12 @@ void BulkController::sendBytes(const QByteArray& data) {
             (unsigned char*)data.constData(),
             data.size(),
             &transferred,
-            0);
+            kBulkSendTimeout);
```

Comment:
```suggestion
            std::milliseconds(kBulkSendTimeout).count();
```

Reply:
What's the aim to do that? Appreciate this is all constexpr and won't have an impact on CPU cycles, but why would we to effectively do a double cast? (int -> std::milliseconds -> int)
If it's for readability, how about I rename `kBulkSendTimeoutMillsecond` or something like this?

Reply:
Ah no, you changed `kBulkSendTimeout` - find then. This still feels redundant IMO, but I'm happy to aim to use as much `std` as possible.

Reply:
well, its about using semantic types. If there is type to represent the data, then prefer to use that type instead of a primitive. Otherwise its too easy to accidentally end up with garbage calculations like `seconds * donuts + earthRadius` (because the semantic type should prevent you trying to do garbage operations like that). 

Fortunately, do to constexpr, the resulting binary should be same.

Reply:
`std::milliseconds(kBulkSendTimeout).count() * donuts + earthRadius` would still be possible.
I get the point, but I'm just saying for a value that gets defined in anonymous namespace and is referenced once as input to a function that wants a `uint`, I don't think it is adding much value to semantic or type safety. If that variable was coming for the function input, he would definitely make sense!

Reply:
> std::milliseconds(kBulkSendTimeout).count() * donuts + earthRadius would still be possible.

Yes, but its much easier to spot that something fishy is going on here.